package.json

```diff
@@ -18,7 +18,8 @@
   "main": "lib/index.js",
   "files": [
     "/lib",
-    "/dist"
+    "/dist",
+    "/src"
```

Comment:
do we need to copy all of the src directory to npm for this?

Reply:
You still need source code with types. Just don't need to copy it to different folder.

Reply:
Oh I see. Hence the importing from `../src`. Does it seem like an antipattern to you for lib to import from src?

Reply:
This is just a trick. Sources are not imported in actual code. Also for me published source is just a way to observe code from node_modules without opening github.